sdk/nodejs/src/client.ts

```diff
@@ -106,23 +106,24 @@ export class DVCClient {
 
     variable<T extends DVCVariableValue>(user: DVCUser, key: string, defaultValue: T): DVCVariable<T> {
         const incomingUser = castIncomingUser(user)
+        // this will throw if type is invalid
```

Comment:
does this mean it will throw the function as well? I thought it was supposed to log an error and just give default instead of throwing an error.

Reply:
this is for when you've passed a default value that contains an invalid type, and yes it will throw